test/integration/query-interface.test.js

```diff
@@ -156,7 +156,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
 
       return Users.sync({ force: true }).then(function() {
         return self.queryInterface.describeTable('_Users', {logging: log}).then(function(metadata) {
-          expect(count).to.be.equal(1);
+          expect(count).to.be.at.least(1);
```

Comment:
I saw in the test that it doesn't count queries for SQLite. There's a comment saying "sqlite fires a lot more querys than the other dbs. this is just a simple hack, since i'm lazy"!

In any case, it seems to me that the counting of how many queries fire is redundant as the tests also correctly test the result of those queries executing (e.g. after calling `dropAllTables()`, it then uses `showAllTables()` to check there are none left).

So I decided, rather than overhaul the tests entirely in this PR, to modify them the minimal amount necessary to make them workable. They were failing without these changes as the tests were written to require the incorrect behaviour passing `options.logging`!

Does this make sense, @BridgeAR?